### PR TITLE
Improved Windows Build Docs

### PR DESCRIPTION
**Pull request by isidorosp**
Added more detailed instructions on order, dependencies, and steps required to build rotki from source on Windows.

### CODE REVIEW

File: docs/installation_guide.rst [hunk 1]
```diff
@@ -169,58 +169,143 @@ You can now start Rotki::
...
+3. To test if you have entered python correctly into the Path variable, open a command prompt and type in ``python`` then hit Enter. The python cli should run and you should see the python version you installed depicted above the prompt. Press CTRL+Z, then Enter to exit.
+
+    .. NOTE::
+        For some reason in newer versions of Windows typing "python" will actual open the Windows Store -- you can fix this by opening "App execution aliases" (search for it via the Windows Search) and toggling off the aliases for python.exe and python3.exe.
```
Comment (LefterisJP) [inline docs/installation_guide.rst]: actual -> actually

File: docs/installation_guide.rst [hunk 2]
```diff
@@ -169,58 +169,143 @@ You can now start Rotki::
...
 
-    library_dirs=[r'C:\Users\yourusername\w\sqlcipher'],
-    include_dirs=[r'C:\Users\yourusername\w'],
+If at any time you want to dissasociate with the virtual env, you can use the command ``deactivate``. Whenever you open a new terminal you can now use ``workon rotki-develop`` (if you named your virtualenv something else then use that instead of ``rotki-develop``) and it should establish the link to the python virtualenv you created and set your working directory to the directory you were in in Step 5. Following the example above, if you open a brand new terminal and type in ``workon rotki-deveop`` your terminal prompt should look something like::
+
+    (rotki-develop) c:\dev\rotki-develop>
+
+6. Now it's time to install all the python requirements. In the open terminal with your virtualenv activated, execute::
+
+    install -r requirements_dev.txt
```
Comment (LefterisJP) [inline docs/installation_guide.rst]: shouldn't this be `pip install`?

File: docs/installation_guide.rst [hunk 3]
```diff
@@ -169,58 +169,143 @@ You can now start Rotki::
...
+    mkvirtualenv rotki-develop
 
-Then you can go to ``pysqlcipher3`` and edit the ``setup.py`` file to include the location of the headers and the libraries you just compiled. Add the following to the extension that is being built.::
+5. Ensure that you're in the directory where you downloaded/cloned the rotki source and bind the virtualenv to the directory::
 
+    setprojectdir .
 
-    library_dirs=[r'C:\Users\yourusername\w\sqlcipher'],
-    include_dirs=[r'C:\Users\yourusername\w'],
+If at any time you want to dissasociate with the virtual env, you can use the command ``deactivate``. Whenever you open a new terminal you can now use ``workon rotki-develop`` (if you named your virtualenv something else then use that instead of ``rotki-develop``) and it should establish the link to the python virtualenv you created and set your working directory to the directory you were in in Step 5. Following the example above, if you open a brand new terminal and type in ``workon rotki-deveop`` your terminal prompt should look something like::
```
Comment (LefterisJP) [inline docs/installation_guide.rst]: workon rotki-deveop -> workon rotki-develop

File: docs/installation_guide.rst [hunk 4]
```diff
@@ -169,58 +169,143 @@ You can now start Rotki::
...
+            name=PACKAGE_NAME + EXTENSION_MODULE_NAME,
+            sources=sources,
+            define_macros=define_macros)
+        ],
+    Lines 165 ... AFTER
+            ext_modules=[Extension(
+            name=PACKAGE_NAME + EXTENSION_MODULE_NAME,
+            sources=sources,
+            library_dirs=[r'<DIRECTORY WHERE YOU BUILT SQLCIPHER TO (i.e. where the compiled sqlcipher.exe and sqlcipher.dll are)>'],
+	        include_dirs=[r'<THE PARENT DIRECTORY OF THE ABOVE DIRECTORY>'],
```
Comment (LefterisJP) [inline docs/installation_guide.rst]: can you make sure the indentation of the `include_dirs` and the `library_dirs` are the same?

File: docs/installation_guide.rst (hunk 1 above)
Comment (isidorosp) [reply docs/installation_guide.rst]: Fixed.

File: docs/installation_guide.rst (hunk 2 above)
Comment (isidorosp) [reply docs/installation_guide.rst]: Indeed.

File: docs/installation_guide.rst (hunk 3 above)
Comment (isidorosp) [reply docs/installation_guide.rst]: Fixed

File: docs/installation_guide.rst (hunk 4 above)
Comment (isidorosp) [reply docs/installation_guide.rst]: Fixed.

**Pull request merged**